packages/odata-v4/src/uri-conversion/get-expand.ts

```diff
@@ -58,7 +58,7 @@ function getExpandAsString<EntityT extends Entity>(
       params = {
         ...params,
         ...createGetFilter(uriConverter).getFilter(
-          and(...expand._filters?.filters),
+          and(expand._filters?.filters || []),
```

Comment:
i assume this is equivalent in the new TS version.

Reply:
The new TS version is stricter.
Since `expand._filters?` can be `undefined`, we might pass `undefined` to `and`, which was wrong.